scripts/migrate_warmboot_database

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/python
+from __future__ import print_function
+import sys
+import swsssdk
+import redis
+import argparse
+
+def migrate(target_dbname):
+    target_dbport = swsssdk.SonicDBConfig.get_port(target_dbname)
+    target_dbhost = swsssdk.SonicDBConfig.get_hostname(target_dbname)
+
+    dblists = swsssdk.SonicDBConfig.get_dblist()
+    for dbname in dblists:
+        dbport = swsssdk.SonicDBConfig.get_port(dbname)
+        # if the db is on the same instance, no need to migrate
+        if dbport == target_dbport:
+            continue
```

Comment:
Is it enough to determine "the same instance" ?

Reply:
good catch. hostname plus port  is enough to determine the "unique instance" . Updated